src/core/exception.d

```diff
@@ -371,6 +371,36 @@ unittest
     }
 }
 
+/**
+ * Thrown on an array misalignment error, that may occur during an array reinterpreting case.
+ */
+class ArrayMisalignmentError : Error
+{
+    @safe pure nothrow @nogc this(string file = __FILE__, size_t line = __LINE__, Throwable next = null )
```

Comment:
attributes should go on the RHS. Also, trailing space between `null` and closing `)`